container-storage-setup.spec

```diff
@@ -15,10 +15,7 @@ URL:            http://github.com/projectatomic/container-storage-setup/
 Source0: %{git0}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
 BuildArch: noarch
 
-BuildRequires:  pkgconfig(systemd)
-
 Requires:       lvm2
-Requires:       systemd-units
```

Comment:
We probably should fix the script to not blow up if /usr/sbin/systemctl does not exist. Most likely if it does not exist, we don't need to run that code.  Since we are only attempting to cleanup some thing that was done for systemd...